README.md

```diff
@@ -1,2 +1,11 @@
-# hw01
```

Comment:
Evaluation HW-1 for Miray Palaz
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     √      |              |            |
| **Coding strategy**         |     √      |              |            |
| **Presentation**            |    √       |              |            |
| **Achievement, creativity** |     √      |              |            |
| **Ease of access**          |    √       |              |            |

It is really good to see someone updating a picture along with the code as I have done such something for my own assignment, and the cat is pretty cute.
Overall the code is written in an excellent quality and fulfill all the given requirement. Just have one question about what is the readme.html file?
Anyway, good job.

Reply:
Evaluation HW-01 for Miray Palaz
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       √    |              |            |
| **Coding strategy**         |   √        |              |            |
| **Presentation**            |     √      |              |            |
| **Achievement, creativity** |      √     |              |            |
| **Ease of access**          |     √      |              |            |

Remarks:

* Hi Miray, I love the pictures and your self-introduction. I really love CATS!!!!! You have included all the elements required with Rmarkdown, which takes me quite some time to figure out how it works.
* The only suggestion I can provide now is to split your first paragraph into different parts so that your logic will be further clarified.
* Overall it's awesome! Keep going!